clients/common/norm.cpp

```diff
@@ -66,16 +66,15 @@ double norm_check_general<rocblas_half>(char norm_type,
                                         rocblas_half* hCPU,
                                         rocblas_half* hGPU)
 {
-    // norm type can be O', 'I', 'F', 'o', 'i', 'f' for one, infinity or Frobenius norm
+    // norm type can be 'O', 'I', 'F', 'o', 'i', 'f' for one, infinity or Frobenius norm
     // one norm is max column sum
     // infinity norm is max row sum
     // Frobenius is l2 norm of matrix entries
 
     double error_double = std::numeric_limits<double>::quiet_NaN();
 
```

Comment:
host_vector<> is a much simpler mechanism than unique_ptr. There is no need to use unique_ptr simple to free data when a variable goes out of scope. Any class with a destructor is sufficient.